src/Model/FileTag.php

```diff
@@ -98,15 +102,19 @@ public static function arrayToFile(array $array, string $type = 'file')
 	 *
 	 * ex. given <music><video>[friends], return [music,video] or [friends]
 	 *
-	 * @param string $file File tags
-	 * @param string $type Optional file type.
+	 * @param string|null $file File tags
+	 * @param string      $type Optional file type.
 	 *
 	 * @return array        List of tag names.
 	 */
-	public static function fileToArray(string $file, string $type = 'file')
+	public static function fileToArray($file, string $type = 'file')
```

Comment:
I'd rather if you left the strict definition of this method and instead changed `include/text.php:189` with:
```php
foreach (FileTag::fileToArray($item['file'] ? '', 'category') as $savedFolderName) {
```
And `include/text.php:205`
```php
foreach (FileTag::fileToArray($item['file'] ?? '') as $savedFolderName) {
```